ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -636,12 +636,21 @@ abstract class QueryStoreAndAuthDependentIntegrationTest
           withBazRecord("c"),
           withBazRecord("a"),
         ),
+        Scenario(
+          "gt string with unsafe values",
+          kbvarId,
+          kbvarVA,
+          Map("bazRecord" -> Map("baz" -> Map("%gt" -> "bobby'); DROP TABLE Students;--")).toJson),
+        )(
+          withBazRecord("c"),
+          withBazRecord("a"),
+        ),
         Scenario(
           "gt int",
           kbvarId,
           kbvarVA,
           Map("fooVariant" -> Map("tag" -> "Bar".toJson, "value" -> Map("%gt" -> 2).toJson).toJson),
-        )(withFooVariant(3), withFooVariant(1)),
+        )(withFooVariant(10), withFooVariant(1)),
```

Comment:
Using 10 rather than 3 verifies that it's using numeric comparisons rather than textual comparisons.